src/gui/viewer.cpp

```diff
@@ -390,6 +394,32 @@ void ViewerMainWindow::reload_file() {
 	document()->load_file(document()->path().c_str());
 }
 
+void ViewerMainWindow::import_file() {
+	auto filename = QFileDialog::getOpenFileName(this, tr("Open Image"), QString(), "JPG Files (*.jpg)").toUtf8();
+
+	//	ImageImportDialog dialog(this);
+	//	dialog.exec();
```

Comment:
These two commented lines can be removed

Reply:
reverted to `zoom`